index.html

```diff
@@ -248,51 +186,89 @@ <h3><dfn>Sampling Frequency</dfn> and <dfn>Reporting Frequency</dfn></h3>
   </p>
 </section>
 
+<section> <h2>Pressure States</h2>
+  <p>
+    <dfn>Pressure states</dfn> represents the minimal set of useful states that allows web applications
+    to react to changes in compute pressure with minimal degration in quality or service, or user
+    experience.
+  </p>
+  <pre class="idl">
+    enum PressureState { "nominal", "fair", "serious", "critical" };
+  </pre>
+  <p>
+    The <dfn>PressureState</dfn> enum represents the [=pressure state=] with the following states:
+  </p>
+  <p>
+    <style>
+      @counter-style pressure-states-emoji {
+        system: cyclic;
+        symbols: "⚪" "🟢" "🟡" "🔴";
+        suffix: " ";
+      }
+      .pressure-states {
+        list-style-type: pressure-states-emoji;
+      }
+    </style>
+    <ul class="pressure-states">
+      <li>
+        <b>Nominal</b>: Work is minimal and the system is running on lower clock speed to preserve power.
```

Comment:
It seems like "nominal" doesn't reflect the description. Should it be "preserve-power"?

Reply:
> it has nothing to be about preserving power, 

But spec says: "Work is minimal and the system is running on lower clock speed to preserve power."
I'm confused...